pymc3/distributions/continuous.py

```diff
@@ -3726,6 +3726,18 @@ def logcdf(value, mu, s):
             0 < s,
         )
 
+class LogitNormalRV(RandomVariable):
+    name = "logit_normal"
+    ndim_supp = 0
+    ndims_params = [0, 0]
+    dtype = "floatX"
+    _print_name = ("logitNormal", "\\operatorname{logitNormal}")
+
+    @classmethod
+    def rng_fn(cls, rng, mu, sigma, size=None):
+        return invlogit(stats.norm.rvs(loc=mu, scale=sigma, size=size, random_state=rng))
```

Comment:
Should use `scipy.expit`. I think `invlogit` is the aesara symbolic counterpart. Here the output it shouldn't be symbolic anymore